eip_operator/src/controller/eip.rs

```diff
@@ -87,6 +88,42 @@ impl k8s_controller::Context for Context {
             }
         };
         crate::eip::set_status_created(&eip_api, name, &allocation_id, &public_ip).await?;
+
+        if eip.status.as_ref().is_some_and(|s| s.resource_id.is_some()) {
+            // do nothing
```

Comment:
can we just invert the condition here instead of having an empty block? this kind of structure is harder to read